pypeit/extraction.py

```diff
@@ -171,7 +168,7 @@ def __init__(self, sciImg, sobjs_obj, spectrograph, par, caliBrate,
 
         # Load up other input items
         self.bkg_redux = bkg_redux
-        self.find_negative = find_negative
+        self.return_negative=return_negative
```

Comment:
is `return_negative` documented somewhere?  If not, please do so.

Reply:
Done.